tests/oeedger8r/edl/deepcopy.edl

```diff
@@ -34,6 +34,13 @@ enclave {
     [count=count, size=size] uint64_t* ptr;
   };
 
+  struct NestedStruct {
```

Comment:
This is the only test for recursive deep-copying (that is, more than the levels supported by Intel's edger8r). While I've not yet committed to supporting recursive deep-copying in v6, it's probably doable. However, it will take extra testing and error checking (as there are a lot more edge cases to consider, especially with respect to cycles).